models/articles.models.js

```diff
@@ -21,3 +15,18 @@ exports.fetchArticleById = (article_id) => {
       return article;
     });
 };
+
+exports.fetchAllArticles = (sort_by = "created_at") => {
+  let queryStr = `SELECT articles.article_id, articles.title,
+    articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, count(comments.comment_id)
+    AS comment_count FROM articles 
```

Comment:
If the comment_count isn't the right data type, you might want to consider using the CAST SQL functionality, which "casts" one datatype to another